src/main/java/com/redhat/documentation/asciidoc/extraction/ExtractedModule.java

```diff
@@ -33,6 +34,12 @@ static boolean isNodeAModule(StructuralNode node) {
         return false;
     }
 
+    static String getFolder(Document doc) {
```

Comment:
Not sure why this is static, shouldn't be. Should also be public. You also shouldn't need to pass anything into it. You have the section which is a `StructuralNode`, you can do the same thing with it for this purpose as a `Document`.